code/modules/admin/verbs/massmodvar.dm

```diff
@@ -64,7 +64,8 @@
 
 	if(default == VV_NUM)
 		var/dir_text = ""
-		if(dir < 0 && dir < 16)
+		var/dir = var_value

```

Comment:
come on you know what's wrong here.

Reply:
locals shouldn't take the name of object vars

Reply:
Oh, ya, I knew that, but I'm lazy af

Reply:
*growl*